plugins/wazo-snom/common/common.py

```diff
@@ -156,6 +157,14 @@ def __init__(self, app, plugin_dir, gen_cfg, spec_cfg):
 
     http_dev_info_extractor = BaseSnomHTTPDeviceInfoExtractor()
 
+    def _add_uxm_firmware(self, raw_config):
+        f = glob.glob(self._tftpboot_dir + u'/firmware/snomD7C-*.bin') + glob.glob(self._tftpboot_dir + u'/firmware/snomUXM-*.bin')
+        if len(f):
```

Comment:
You can just do `if f:` - an empty list is false-y

Reply:
done !